.github/workflows/ci.yml

```diff
@@ -50,4 +50,4 @@ jobs:
         with:
           key: tests
       - name: Run test
-        run: pixi run test --color always${{ startsWith(matrix.os, 'ubuntu') && ' --no-default-features --features rustls-tls' || '' }}
+        run: pixi run test --color always${{ startsWith(matrix.os, 'ubuntu') && ' --no-default-features --features rustls-tls' || '' }} -- --test-threads=4
```

Comment:
I edited CI script to reduce parallel test, or it would make too many requests to download packages at the same time.